data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+		angle -15
+		zoom 0.5
+	"reverse engine" 9 -15
+		angle 15
+		zoom 0.5
+	"steering engine" -9 -15
+		angle 15
+		zoom 0.5
+		right
+	"steering engine" 9 -15
+		angle -15
+		zoom 0.5
+		left
+	"steering engine" -9 -15
+		angle 165
+		zoom 0.5
+		left
+	"steering engine" 9 -15
+		angle 195
+		zoom 0.5
+		right
+
+	gun -6 -30
+	gun 0 -30
+	gun 6 -30
+	turret 0 -33 "Speckle Turret"
+
+	leak "flame" 30 60
+	explode "tiny explosion" 15
+	"final explode" "final explosion small"
+
+	description "The Photikon Torpedo Bomber is the cornerstone of Avgi antiship carrier doctrine. While it is possible for a swarm of Vibratia and similar interdiction drones to engage capital ships, it is the Photikon that is capable of delivering the nuclear anti-ship munitions necessary to bring down the largest warships. By bringing a nuclear torpedo within close range, the payload has a higher chance of surviving enemy point defense weapons compared with standard missiles."
+	description "Photikons are occasionally used as gun platforms, mounting a Speckle Turret instead of their standard armament of two Ophrys Torpedoes."
+	description "	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay."
+
+
+ship "Photikon" "Photikon (Atomic)"
+	outfits
+		"External Launch Rail" 3
+		`"Ophrys" Nuclear Torpedo` 3
+		
+		"Concentrating Solar Array"
+		"Ultracapacitor Cell" 3
+		"Multispectral Scanner"
+		
+		"Magnetoplasma Drive"
+
+
+# Drones
+
+ship "Vibratia"
+	sprite "ship/avgi vibratia"
+	thumbnail "thumbnail/avgi vibratia"
+	attributes
+		category "Drone"
+		licenses
+			"Twilight Guard"
+		"cost" 96000
+		"hull" 1500
+		"automaton" 1
+		"crew equivalent" 1
+		"threshold percentage" 0.12
+		"mass" 24
+		"drag" 0.3
+		"heat dissipation" 0.99
+		"fuel capacity" 60
+		"outfit space" 75
+		"weapon capacity" 24
+		"engine capacity" 24
+		"radar jamming" 3
+		weapon
+			"blast radius" 12
+			"shield damage" 240
+			"hull damage" 150
+			"hit force" 120
+	outfits
+		"Speckle Coilgun"
+		"Speck Round" 900
+	
+		"Concentrating Solar Array"
+		"Ultracapacitor Cell"
+		"Multispectral Scanner"
+	
+		"Magnetoplasma Drive"
+
+	engine 0 27
+	"reverse engine" -5 -12
+		zoom 0.5
+	"reverse engine" 5 -12
+		zoom 0.5
+	"steering engine" -5 -12
+		angle 90
+		zoom 0.5
+		right
+	"steering engine" 5 -12
+		angle -90
+		zoom 0.5
+		left
+	"steering engine" -3 24
+		angle 90
+		zoom 0.5
+		left
+	"steering engine" 3 24
+		angle -90
+		zoom 0.5
+		right
+	gun 0 -27 "Speckle Coilgun"
+	gun 0 -27
+
+	explode "tiny explosion" 9
+
+	description "The Vibratia Interdictor is the primary Avgi strike fighter. While individually fragile, Vibratia are usually deployed in large swarms, and are agile enough to be surprisingly survivable even under heavy fire."
+	description "	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay."
+
+
+
+ship "Undulon"
```

Comment:
The Undulon is extremely slow, has moderate HP for a T1 drone, and has only a single gun mount, all in exchange for having far more weapon space than any other drone in the game. I do think that in this case the downsides are sufficient to balance it out, particularly given that the limited outfit space effectively restricts it to only missile weapons, and the low speed limits its ability to rearm from a carrier during combat.